tests/drivers/nrfx_integration_test/prj.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+CONFIG_ASSERT=y
+CONFIG_BT=y
+CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
```

Comment:
I think you might need adding `CONFIG_ZTEST=y` (https://docs.zephyrproject.org/latest/guides/test/ztest.html#quick-start-integration-testing)